save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/contests/ContestSummaryMenu.kt

```diff
@@ -43,7 +43,7 @@ private fun ChildrenBuilder.displayTopProjects(sortedResults: List<ContestResult
                 "${index + 1}. ",
                 contestResult.projectName,
                 contestResult.organizationName,
-                contestResult.score?.toString() ?: "-",
+                (contestResult.score?.asDynamic().toFixed(2) as String?) ?: "-",
```

Comment:
We already have an extension for it: https://github.com/saveourtool/save-cloud/blob/11117025df8326159aeb35c61f176dab51b11971/save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/Utils.kt#L147

Reply:
https://github.com/saveourtool/save-cloud/pull/1431